README.md

```diff
@@ -1,9 +1,18 @@
+
+### Firmware Packages & dentOS Tool Versions
 Packages for Mellanox and Marvell Firmware and specific versions of tools used to build dentOS
 
-mstpd
-ethtool 5.6.0
-iproute2 5.6.0
-lldpd 1.0.5
-mft 4.14.0-105
-realpath 8.26-3
-frr 6.0.3
+- mstpd
+- ethtool 5.6.0
+- iproute2 5.6.0
+- lldpd 1.0.5
+- mft 4.14.0-105
+- realpath 8.26-3
+- frr 6.0.3
+
+---
+### Dent GNS3 Appliance
+
+All necessary files for setting up the DENT GNS3 Appliance, including the disk image, XML configuration, and related artifacts can be found here: [DENT GNS3 Appliance](https://1drv.ms/f/c/b4d5fd54a1a7d444/EkTUp6FU_dUggLQdrwAAAAABU-4TzozwwGtsV7u5ib1Q0A?e=mEEETi)
+
+For information on how to set up the DENT GNS3 Appliance in GNS3 please view the [DENT DOCS](https://docs.dent.dev/Installation.html)
```

Comment:
1.  I don’t think we want to link to some third party cloud share. No idea about alternatives. Maybe the DENT infrastructure.
2.  It’d be great if you elaborated as you did in the answer to my comment.
3.  Lastly, an additional guide is needed how the image was produced, so that your steps are reproducible.

Reply:
1. Members of the DENT community are actively discussing alternative hosting locations. 
2.  I have elaborated on the hosting concern in the answer to your comment.
3. I will reach out and see if I can provide more information on how the image was produced

Reply:
@paulmenzel I have added a guide in the comments section to provide information on how the image was produced and so that the steps are reproducible. Let me know if you would like me to clarify anything. I appreciate your feedback.